src/client/dfuse/dfuse_main.c

```diff
@@ -242,6 +242,7 @@ show_help(char *name)
 		"	-S --singlethread	Single threaded\n"
 		"	-t --thread-count=count	Number of fuse threads to use\n"
 		"	-f --foreground		Run in foreground\n"
+		"	   --enabling-caching	Enable all caching\n"
```

Comment:
should it not be "--enable-caching" ??

Reply:
You are correct. I'm not sure how that happened, but it is now fixed. Thanks.